python/mxnet/text/embedding.py

```diff
@@ -496,44 +503,27 @@ class GloVe(TokenEmbedding):
     """
 
     # Map a pre-trained token embedding archive file and its SHA-1 hash.
-    pretrained_archive_name_sha1 = \
-        {'glove.42B.300d.zip': 'f8e722b39578f776927465b71b231bae2ae8776a',
-         'glove.6B.zip': 'b64e54f1877d2f735bdd000c1d7d771e25c7dfdc',
-         'glove.840B.300d.zip': '8084fbacc2dee3b1fd1ca4cc534cbfff3519ed0d',
-         'glove.twitter.27B.zip': 'dce69c404025a8312c323197347695e81fd529fc'}
+    pretrained_archive_name_sha1 = C.GLOVE_PRETRAINED_FILE_SHA1
```

Comment:
We can replace pretrained_archive_name_sha1 with C.GLOVE_PRETRAINED_FILE_SHA1. There is no need for an extra var called pretrained_archive_name_sha1.

Reply:
Isn’t that the interface? Remember hasattr?